dev/readthedocs.md

```diff
@@ -5,27 +5,55 @@ answer the following questions.
 
 1. What is the command I run to get the courses from Fall 2021?
 
+    `python manage.py ingest jhu --years 2021 --terms Fall`
+
 2. How do I then load those courses into my database?
 
+    `python manage.py digest jhu`
+
 3. How do I get a terminal running in my docker container?
 
+    Opening the Docker explorer shoud show a list of three containers. Right clicking any of these should bring up the `Attach Shell` option. Selecting this option opens a terminal to the corresponding container.
+
 4. Where do I store data such as passwords or secrets that I don’t want to commit?
 
+    `semesterly/sensitive.py`
+
 5. What is our stack?
 
+    | Component          | Technology  | Style/Methodology |
+    |--------------------|-------------|-------------------|
+    | Database           | PostgreSQL  | Django ORM        |
+    | Backend Framework  | Django      | pycodestyle/Black |
+    | Frontend Framework | React/Redux | ESLint/Prettier   |
+    | CSS Framework      | SCSS        | BEM/Airbnb        |
```

Comment:
Lol copied this straight for the rst file didn't you

Reply:
yessir